BUILD

```diff
@@ -13,3 +13,12 @@ filegroup(
         "//os:srcs",
     ],
 )
+
+# For use in Incompatible Target Skipping:
+# https://docs.bazel.build/versions/master/platforms.html#skipping-incompatible-targets
```

Comment:
Would you also update the documentation to reference this? i.e. the `:not_compatible` references in https://docs.bazel.build/versions/master/platforms.html#more-expressive-constraints?

Reply:
Also, in addition to the link could you include a concise sentence describing what this is?

Something like "Lets targets declare incompatibility with some set of platforms. See [link]".

Reply:
Thanks for the suggestion. I largely adopted that.